src/test/scala/org/zalando/nakadi/client/KlientSpec.scala

```diff
@@ -74,44 +44,28 @@ class KlientSpec extends WordSpec with Matchers with BeforeAndAfterEach with Laz
     }
   }
 
-  private def performStandardRequestChecks(expectedRequestPath: String, expectedRequestMethod: HttpString) = {
+  private def performStandardRequestChecks(expectedRequestPath: String, expectedRequestMethod: HttpString): NakadiRequest = {
```

Comment:
I agree. I usually do this with public functions. Sometimes , I do it for private functions as well..depends on the context but I am fine with always specifying it